lib/ssh/test/ssh_key_cb.erl

```diff
@@ -0,0 +1,45 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2004-2015. All Rights Reserved.
```

Comment:
This should only list 2015, since its created now. As soon as that's updated here and in  lib/ssh/test/ssh_key_cb_options.erl I'll see if we can merge this :)